src/features/assets/index.ts

```diff
@@ -0,0 +1,7 @@
+export function getAssetDomain(assetId: string) {
+  return assetId.split('@')[1];
+}
+
+export function getAssetName(assetId: string) {
+  return assetId.split('#')[0];
+}
```

Comment:
Actually, it might be useful to introduce classes for some IDs.

Backend gives IDs in a format Iroha supports it:

- `signatory@domain` for accounts
- `name#domain` for asset _definitions_
- `asset#asset_domain#signatory@account_domain` for assets (`asset##signatory@domain` - when both the asset definition and the account are in the same domain)

From frontend side, it is sometimes needed to display it in an original form, but sometimes it is necessary to decompose/compose back.

So, I propose to introduce classes `AccountId`, `AssetDefinitionId`, and `AssetId` and integrate them with Zod.

They might look something like this:

```ts
class AssetDefinitionId {
    public construction(name: string, domain: string) {
        // ...
    }

    public get name(): string {}

    public get domain(): string {}

    public toString(): string {} // back to `name#domain`
}

const assetDefinitionSchema = z.string().transform((ctx) => {
    // parse, through error via ctx
    const name, domain

    return new AssetDefinitionId(name, domain)
})
```

Reply:
For now I use only AssetDefinitionId and AssetId. AccountId will be implemented later if needed.